### ISSUE TITLE
Autoprefixer as a plugin stopped working with the new version of rollup-plugin-postcss

### ISSUE THREAD

**Issue by David-Else** [egoist/rollup-plugin-postcss#57]
Hello,

I don't know if it's the recent update, or me changing something accidentally that has caused this error, but autoprefxer now does nothing when it used to work! 

If i set `minimize: true` if minifies OK. Has something broken in the new version, or changed with the API?

I took the last lines of CSS in the source file from https://autoprefixer.github.io/ , it should add prefixes at any default setting.

Here are some details of my settings, thanks for your help!:

** EDIT I just saw that there were breaking API changes using the `node-update` utility, but the new 1.0+ docs on the front page don't really explain how to use any other plugins and gives no examples like the old front page used to. I still have no idea how to make autoprefixer work, maybe I am stupid? :)

EDIT 2 - Just saw your Twitter, THANKS for this amazing application and your continued support! I really love it, I spent the day looking into switching to webpack, and it was dependency hell. I hate how bloated JS development has become, every extra package in the tool-chain is another point of failure, I am not about to install 5,000 NPM packages by 100 different people just to print my name on the screen in JS :) I am spending more time with tooling than programming, I am hoping this will all end with my 'final' Rollup set-up.

```
**package.json**

"devDependencies": {
    "autoprefixer": "^7.2.4",
    "rollup": "^0.53.4",
    "rollup-plugin-postcss": "^1.0.2"
  }
```
```
**rollup.config.js**

import postcss from 'rollup-plugin-postcss';
import autoprefixer from 'autoprefixer';

export default {
  input: 'src/main.js',
  output: {
    file: 'dist/bundle.js',
    format: 'iife',
    banner: '/* http://www.elsewebdevelopment.com/ */',
  },
  plugins: [
    postcss({
      plugins: [
        autoprefixer(),
      ],
      sourceMap: true,
      extract: true,
      extensions: ['.css'],
      // minimize: true,
    }),
  ],
  sourcemap: true,
};
```
```
**The source file**

* {
    box-sizing: border-box;
}

#canvas {
    outline: black 2px solid;
}

.example {
    display: flex;
    transition: all .5s;
    user-select: none;
    background: linear-gradient(to bottom, white, black);
}
```
```
**the destination file**

* {
    box-sizing: border-box;
}

#canvas {
    outline: black 2px solid;
}

.example {
    display: flex;
    transition: all .5s;
    user-select: none;
    background: linear-gradient(to bottom, white, black);
}

/*# sourceMappingURL=bundle.css.map */
```

**Comment by egoist** [comment]
Fixed in `v1.0.3`, btw `extensions` option is no longer needed.

**Comment by BiYuqi** [comment]
Can you given the doc the config about how to use autoprefixer.? now It is difficult to set up. no friendly to usage, thank you.
